vernac/vernacentries.ml

```diff
@@ -2237,12 +2256,14 @@ let interp ?(verbosely=true) ?proof st (loc,c) =
 	    ignore (Flags.use_polymorphic_flag ());
             iraise e
   in
-    if verbosely then Flags.verbosely (aux false) c
-    else aux false c
+  let atts = { loc; locality = None; polymorphic = false; program_mode = orig_program_mode } in
```

Comment:
I didn't find the time to check it yet, but doesn't this make the following script error?
```
Set Program Mode.
Program Definition id (n : nat) := n.
```

Reply:
Let me have a look...

Reply:
Oh indeed, what a headache the program flags are [I regret adding that last commit XD]